specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/webapplicationfirewall.json

```diff
@@ -453,13 +453,28 @@
           "type": "boolean",
           "description": "Whether to allow WAF to check request Body."
         },
+        "reqBodyInspectLimitInKB": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 8,
+          "exclusiveMinimum": false,
+          "description": "Max inspection limit in KB for request body inspection for WAF."
+        },
+        "requestBodyEnforcement": {
+          "type": "boolean",
+          "description": "Whether allow WAF to enforce request Body limits."
+        },
         "maxRequestBodySizeInKb": {
           "type": "integer",
           "format": "int32",
           "minimum": 8,
           "exclusiveMinimum": false,
           "description": "Maximum request body size in Kb for WAF."
         },
+        "fileUploadEnforcement": {
+          "type": "boolean",
+          "description": "Whether allow WAF to enforce file upload limits."
+        },
         "fileUploadLimitInMb": {
```

Comment:
Not sure if this is already discussed. In older APIs, this is enabled by default (since there was no configuration).
Curious to understand, why we have it disabled by default in the new API, instead of having it enabled by default?

Reply:
done... taken care by default : true behavior.